editor/editor_feature_profile.cpp

```diff
@@ -42,7 +42,7 @@ const char *EditorFeatureProfile::feature_names[FEATURE_MAX] = {
 	TTRC("Scene Tree Editing"),
 	TTRC("Import Dock"),
 	TTRC("Node Dock"),
-	TTRC("Filesystem Dock")
+	TTRC("FileSystem and Import Docks")
```

Comment:
You changed the name, but this checkbox doesn't actually control the existence of the Import dock. I fixed this in #38580.